moduleframework/module_framework.py

```diff
@@ -104,6 +104,18 @@ def runHost(self, command="ls /", **kwargs):
                 trans_dict)
         return process.run("%s" % formattedcommand, **kwargs)
 
+    def PackagesInstalled(self,packagelist, func="runHost"):
```

Comment:
Because it should speedup testing, but this caused opposite effect, so that I've let function there, for further work, but I've let it empty.